emscripten.py

```diff
@@ -1699,6 +1699,10 @@ def create_receiving(function_table_data, function_tables_defs, exported_impleme
 
   shared.Settings.MODULE_EXPORTS = module_exports = exported_implemented_functions + function_tables(function_table_data)
 
+  # Should only need to do this in the main thread.
```

Comment:
Please clarify this in the comment. I believe the issue is that when assertions are on, we emit code that is incompatible with ES6 currently? Can add a TODO in the comment to fix that.

Reply:
Good point, I missed the comment what I changed the behaviour.